tests/e2e/conversion-webhook-invalid-config/00-assert-created.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: opentelemetry.io/v1alpha1
```

Comment:
does  v1beta1 fail on the admision with this config?

Reply:
Yes, since `Config` changed from string to an object:
```
Error from server: error when creating "otel-test.yaml": admission webhook "mopentelemetrycollectorbeta.kb.io" denied the request: json: cannot unmarshal string into Go struct field OpenTelemetryCollectorSpec.spec.config of type v1beta1.Config
```